hack/helm/release/internal/helm/helm.go

```diff
@@ -479,6 +479,8 @@ func copyChartToGCSBucket(ctx context.Context, config copyChartToBucketConfig) e
 	if len(files) == 0 {
 		return fmt.Errorf("couldn't file helm package with glob (%s)", config.sourceGlob)
 	}
+	isSnapshot := strings.Contains(files[0], "SNAPSHOT")
```

Comment:
I'm a little paranoid and so this test seems a little weak to me, but in this context it should be more than enough.

Reply:
Yeah, in this block we only have access to the tarball, so it would be a bit more work to try and get access to the Chart.yaml and check if the `version` field ends in `-SNAPSHOT`. This seems sufficient for our use-case.

I've also updated the comments on this func, as they were incorrect with this current change, and will merged when CI is complete.

Reply:
We could search for `-SNAPSHOT.tgz` :man_shrugging:.

Reply:
Easy enough. Updated.